diff -urN pykde4-4.11.2/CMakeLists.txt pykde4-4.11.2-patched/CMakeLists.txt
--- pykde4-4.11.2/CMakeLists.txt	2013-09-27 06:19:30.000000000 +1100
+++ pykde4-4.11.2-patched/CMakeLists.txt	2013-06-29 04:16:03.000000000 +1100
@@ -114,20 +114,20 @@
 if (PYTHON_VERSION_MAJOR GREATER 2)
     if(PYQT4_VERSION STRGREATER "040904")
 	    # Disable for newer PyQt
-	    set(SIP_EXTRA_OPTIONS -P -g -x PyKDE_QVector)
+	    set(SIP_EXTRA_OPTIONS -g -x PyKDE_QVector)
     else ()
 	    set(SIP_EXTRA_OPTIONS -g)
     endif()
 else (PYTHON_VERSION_MAJOR GREATER 2)
     if(PYQT4_VERSION STRGREATER "040904")
 	# Disable for newer PyQt
-    	set(SIP_EXTRA_OPTIONS -P -g -x PyKDE_QVector -x Py_v3)
+    	set(SIP_EXTRA_OPTIONS -g -x PyKDE_QVector -x Py_v3)
     else ()
 	    set(SIP_EXTRA_OPTIONS -g -x Py_v3)
     endif()
 endif ()
 
-add_definitions(-D_REENTRANT -DQT_CORE_LIB -DQT_GUI_LIB -DUSING_SOPRANO_NRLMODEL_UNSTABLE_API -DSIP_PROTECTED_IS_PUBLIC -Dprotected=public)
+add_definitions(-D_REENTRANT -DQT_CORE_LIB -DQT_GUI_LIB -DUSING_SOPRANO_NRLMODEL_UNSTABLE_API)
 
 file(GLOB kdecore_files_sip sip/kdecore/*.sip)
 set(SIP_EXTRA_FILES_DEPEND ${kdecore_files_sip})
